Makefile

```diff
@@ -21,14 +21,6 @@ all: openconfig_public protos validate_paths
 openconfig_public:
```

Comment:
We can also remove all the references to `openconfig_public` in the Makefile because we only pulled in the OC models to build feature.textproto data.

The file `tools/clone_oc_public.sh` isn't needed either.

Reply:
Done